disk/tests/test_unit.py

```diff
@@ -218,3 +219,19 @@ def test_get_devices_label():
     ):
         labels = c._get_devices_label()
         assert labels.get("/dev/mapper/vagrant--vg-root") == "label:DATA"
+
+def test_min_disk_size():
```

Comment:
For tests you need to pass the aggregator fixture here:
```suggestion
def test_min_disk_size(aggregator):
```

And then use it (like in other tests) to assert metrics and service checks